packages/trace-viewer/src/ui/modelUtil.ts

```diff
@@ -128,16 +130,19 @@ export class MultiTraceModel {
   }
 
   private _errorDescriptorsFromTestRunner(): ErrorDescription[] {
-    const errors: ErrorDescription[] = [];
-    for (const error of this.errors || []) {
-      if (!error.message)
-        continue;
-      errors.push({
-        stack: error.stack,
-        message: error.message
-      });
+    const errorPrompts: string[] = [];
+    for (const action of this.actions) {
+      for (const attachment of action.attachments ?? []) {
+        if (attachment.name === 'errorPrompt')
+          errorPrompts.push(attachmentURL({ ...attachment, traceUrl: action.context.traceUrl }));
+      }
     }
-    return errors;
+
+    return this.errors.filter(e => !!e.message).map((error, i) => ({
```

Comment:
Same assumption here, don't make it even if you think it is technically correct.

Reply:
i moved the error index into the attachment name - that will work as long as we maintain the error index, and I think we do.